doc/getting_started/installation_linux.rst

```diff
@@ -10,8 +10,10 @@ applications on the following Linux distributions:
 
 * Ubuntu 16.04 LTS 64-bit
 * Fedora 25 64-bit
+* Clear Linux
```

Comment:
You mention a version and bit-size for the other distros, but not for Clear Linux?

Reply:
Clear Linux is 64-bit only, and is on rolling releases (e.g. updates 2+ times a day). So there is no point release as in other distros.